schemas/logs/aws/cloud_trail.yml

```diff
@@ -0,0 +1,187 @@
+  type: string
+- name: awsRegion
+  required: true
+  description: The AWS region that the request was made to, such as us-east-2.
+  type: string
+- name: errorCode
+  description: The AWS service error if the request returns an error.
+  type: string
+- name: errorMessage
+  description: If the request returns an error, the description of the error. This message includes messages for authorization failures. CloudTrail captures the message logged by the service in its exception handling.
+  type: string
+- name: eventID
+  required: true
+  description: GUID generated by CloudTrail to uniquely identify each event. You can use this value to identify a single event. For example, you can use the ID as a primary key to retrieve log data from a searchable database.
+  type: string
+- name: eventName
+  required: true
+  description: The requested action, which is one of the actions in the API for that service.
+  type: string
+- name: eventSource
+  required: true
+  description: The service that the request was made to. This name is typically a short form of the service name without spaces plus .amazonaws.com.
+  type: string
+- name: eventTime
+  required: true
+  description: The date and time the request was made, in coordinated universal time (UTC).
+  type: timestamp
+  timeFormat: rfc3339
+  isEventTime: true
+- name: eventType
+  required: true
+  description: 'Identifies the type of event that generated the event record. This can be the one of the following values: AwsApiCall, AwsServiceEvent, AwsConsoleSignIn'
+  type: string
+- name: eventVersion
+  required: true
+  description: The version of the log event format.
+  type: string
+- name: managementEvent
+  description: 'A Boolean value that identifies whether the event is a management event. managementEvent is shown in an event record if eventVersion is 1.06 or higher, and the event type is one of the following: AwsApiCall, AwsConsoleAction, AwsConsoleSignIn,  AwsServiceEvent'
+  type: boolean
+- name: readOnly
+  description: Identifies whether this operation is a read-only operation.
+  type: boolean
+- name: recipientAccountId
+  description: Represents the account ID that received this event. The recipientAccountID may be different from the CloudTrail userIdentity Element accountId. This can occur in cross-account resource access.
+  type: string
+  indicators:
+  - aws_account_id
+- name: requestID
+  description: The value that identifies the request. The service being called generates this value.
+  type: string
+- name: requestParameters
+  description: The parameters, if any, that were sent with the request. These parameters are documented in the API reference documentation for the appropriate AWS service.
+  type: json
+- name: resources
+  description: A list of resources accessed in the event.
+  type: array
+  element:
+    type: object
+    fields:
+    - name: arn
+      description: ARN field null.String
```

Comment:
`null.String`?

Reply:
It's an auto-generated field description as generated by the `mage build:schemas` command. We can discuss how to handle auto-descriptions and make a PR on `panther` repo, re-export and merge here.

Reply:
If the field has no description it uses `{FIELD_NAME} field {FIELD_TYPE}`

Reply:
> If the field has no description it uses `{FIELD_NAME} field {FIELD_TYPE}`

I think it could be just `{FIELD_NAME} field`. The type of it is anyway already present in the field schema, and we can avoid awkward descriptions like this one.

Reply:
I updated the exporting function in the pending PR on `panther` repo. Will dump again and update the schema YAMLs today. Have some more things to tweak in the export code.

Reply:
> I updated the exporting function in the pending PR on `panther` repo. Will dump again and update the schema YAMLs today. Have some more things to tweak in the export code.

@alxarch was this done?